base/printers.go

```diff
@@ -55,17 +55,24 @@ type TablePrinter struct{}
 func (pr *TablePrinter) PrintStream(ctx context.Context, w io.Writer, rp output.RowProducer) error {
 	mc, _ := consolesize.GetConsoleSize()
 	if mc <= 0 {
```

Comment:
it cannot be lower than zero that GetConsoleSize returns uint16 for both return values.

Reply:
> At least we can reduce them to a single if block: if mc == 0
``
	if mc <= 0 {
		mc = tableMaxcols()
	}
	if mc < 1 {
		mc = 1000
	}
```
`TableMaxCols` may also return `0`, so need to check the value of `mc` again.